convert/testdata/threeWayFaToVcf/expected.vcf

```diff
@@ -0,0 +1,3 @@
+chr1	3	.	A	C	100	PASS	.	GT	1/1
```

Comment:
Would you want to add a test for diff alt, producing something like "A     C,G"?

Reply:
Done in next commit